chain/network-primitives/src/lib.rs

```diff
@@ -2,4 +2,5 @@ mod blacklist;
 pub(crate) mod config;
 pub(crate) mod config_json;
 mod network_protocol;
+pub mod time;
```

Comment:
Let's add a comment at `core/primitives/src/time.rs` pointing at this new module? We probably should work on actively removing the old one. I wonder if *I* should work on that?

Reply:
done